src/vs/workbench/contrib/editSessions/browser/editSessions.contribution.ts

```diff
@@ -502,7 +505,10 @@ export class EditSessionsContribution extends Disposable implements IWorkbenchCo
 	private async pickContinueEditSessionDestination(): Promise<URI | undefined> {
 		const quickPick = this.quickInputService.createQuickPick<ContinueEditSessionItem>();
 
-		quickPick.title = localize('continueEditSessionPick.title', 'Continue Edit Session...');
+		const workspaceContext = this.contextService.getWorkbenchState() === WorkbenchState.FOLDER
+			? this.contextService.getWorkspace().folders[0].name
+			: this.contextService.getWorkspace().folders.map((folder) => folder.name).join(', ');
+		quickPick.title = localize('continueEditSessionPick.title', "Continue {0} on", `'${workspaceContext}'`);
```

Comment:
Would this text essentially say "Continue ${foldernames} on"? I wonder whether that might confuse users and whether it could get too long if there's a long list of folders.

Reply:
Yeah this is a new problem and I'm not totally sure what we should do here. The existing Continue Working On implementation doesn't run into this at all because it assumes a single-repo world, but moving forward we are likely to support multi root workspaces with RemoteHub in vscode.dev (in fact you can already build multi root workspaces due to https://github.com/microsoft/vscode/issues/144702#issuecomment-1150487115). Filed https://github.com/microsoft/vscode/issues/158422 to track this